src/main/java/cloud/eppo/BaseEppoClient.java

```diff
@@ -49,6 +48,28 @@ protected BaseEppoClient(
       BanditLogger banditLogger,
       boolean isGracefulMode,
       boolean expectObfuscatedConfig) {
+    this(
+        apiKey,
+        sdkName,
+        sdkVersion,
+        host,
+        assignmentLogger,
+        banditLogger,
+        isGracefulMode,
+        expectObfuscatedConfig,
+        null);
+  }
+
+  protected BaseEppoClient(
+      String apiKey,
+      String sdkName,
+      String sdkVersion,
+      String host,
+      AssignmentLogger assignmentLogger,
+      BanditLogger banditLogger,
+      boolean isGracefulMode,
+      boolean expectObfuscatedConfig,
+      Configuration initialConfiguration) {
```

Comment:
Wondering if we should call this class `FlagConfiguration` and parameter `initialFlagConfiguration`, just to disambiguate from other types of configuration (e.g., the config of the SDK with API key, name, version)

Update: ok I see now have we `ConfigurationStore` and whatnot and configuration applies to flags so I think we're good after all 👌

Reply:
yeah, `Configuration` also includes bandit models and maybe more in the future